src/components/molecules/PlanRoom/Board/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { Box, Image } from '@chakra-ui/react';
+import { DndProvider } from 'react-dnd';
+import { HTML5Backend } from 'react-dnd-html5-backend';
+import Items from 'components/molecules/PlanRoom/RoomTables/Table';
+import logo from 'components/molecules/PlanRoom/IMG/backGroundBoard.png';
+
+const Board = () => (
+  <Box position="relative" id="board" h="400px" w="100%" border="1px solid black">
+    <Image src={logo} alt="Segun Adebayo" zIndex="0" position="absolute" h="398px" w="100%" />
```

Comment:
This `alt` property should probably be empty. ;)

Reply:
Using `absolute` positioning and `height` for board is not a good idea. 
It should be responsive as other components.

Reply:
I did this because the background would not come in correctly on the Box. I knew there was a good path, nevertheless it was not working. I found a way to just replace it. As for the alternative text, it will be corrected.ID referred to another issue that I have already removed from the code. ID will also be removed.